.github/workflows/telemetry.yml

```
@@ -0,0 +1,2 @@
+- name: Workflow Telemetry
```

actionlint (1.7.9)

"jobs" section is missing in workflow

(syntax-check)